package.json

```diff
@@ -28,8 +28,8 @@
   },
   "homepage": "https://github.com/ionic-team/ionic-pwa-elements",
   "devDependencies": {
-    "@stencil/core": "^1.8.11",
-    "np": "^6.5.0",
-    "workbox-build": "^4.3.1"
+    "@stencil/core": "^3.4.0",
+    "np": "^8.0.4",
```

Comment:
Passing thru - with `np` v8+, you'll have to add an `engines` field to `package.json`. Otherwise, the publish will fail

Reply:
Thanks for catching this!

Addressed here: https://github.com/ionic-team/pwa-elements/pull/118/commits/1732d81d6530014039fbc7fbbebf2f8be6b515d7

I'm not wedded to the values. Just tried to pick reasonable defaults.